### PR TITLE
Add FR spellcheck + Adjust site.baseurl

### PR DESCRIPTION
**Pull request by gabrielcossette**
1. Add FR spellcheck
2. Enable spellcheck of Blog Posts (+ adjustment of content)
3. Changed content references of **{{site.baseurl}}** to **/ITStrategy** as some paths were detected by spellchecking. We can easily change again if we have a different URL in the future. Also, it is easier to remember.

### CODE REVIEW

File: _pages/en/employee-profiles.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ These profiles are to help categorize employees and determine their IT and busin
 - Client services - Work directly with the public
 - Call Center User - Similar to an activity-based worker but requires additional telephony options
 - General GC Office users - Usually at their desk
-- Special needs/Accessible - worker with disabilities in the workplace who requires a client’s need assessment and awareness services   for adaptive technology
+- Special needs/Accessible - worker with disabilities in the workplace who requires a client's need assessment and awareness services   for adaptive technology
```
Comment (smellems) [inline _pages/en/employee-profiles.md]: I fully approve changing all ` to ' :confetti_ball:

Comment (gabrielcossette) [reply _pages/en/employee-profiles.md]: Even though I believe ` is valid in English, it wasn't recognized by the spellchecker, thus generating false positives.

**Comment by gabrielcossette** [comment]
> 
> 
> Adding French spellcheck is a good idea. Also enabling cspell for Posts.
> 
> But I still think we should keep the site.baseurl by dropping [42f6752](https://github.com/sara-sabr/ITStrategy/commit/42f6752bbb618f276f159dea328eae96c91d5c9d)
> 
> The errors with the spellcheck come from image file names stored in assets. We could change file names or add the words to cspell. There's also probably a way to tell cspell to ignore regular expressions..
> image

@smellems Sure, I'm open to these 3 options. Which one do you recommend?

**Comment by smellems** [comment]
> @smellems Sure, I'm open to these 3 options. Which one do you recommend?

I had done it here https://github.com/sara-sabr/ITStrategy/pull/523/commits/70d666b6bf019f7c5962c625275c52f13286420e in PR #523 where most of these same changes are already.  Just added the words to cspell, probably the simplest way

Can you also cleanup #523 so we don't have the same changes in multiple PRs open at the same time.  I noticed the gitignore changes are also in #532

**Comment by gabrielcossette** [comment]
> 
> 
> > @smellems Sure, I'm open to these 3 options. Which one do you recommend?
> 
> I had done it here [70d666b](https://github.com/sara-sabr/ITStrategy/commit/70d666b6bf019f7c5962c625275c52f13286420e) in PR #523 where most of these same changes are already. Just added the words to cspell, probably the simplest way
> 
> Can you also cleanup #523 so we don't have the same changes in multiple PRs open at the same time. I noticed the gitignore changes are also in #532

Done. PR #523 and #532 have been cleaned up. This PR should be the only one with changes to cspell and .gitignore.

**Comment by smellems** [comment]
Je pense qu'on devrait aussi enlever les modifications aux variables `site.baseurl`

**Comment by gabrielcossette** [comment]
> 
> 
> Je pense qu'on devrait aussi enlever les modifications aux variables `site.baseurl`

Bon point, c'est fait! :)

**Comment by gcharest** [comment]
Le build ne passe pas parce que le URL joint `l` et `amélioration` en un seul mot... ça risque d'être un problème récurrent en français...

On peut ajouter chaque terme combiné dans les exceptions (un peu comme les acronymes en Anglais) mais on ouvre la porte à des coquilles dans le corps du texte.

**Comment by gcharest** [comment]
Faudrait plutôt utiliser les RegExp

https://www.npmjs.com/package/cspell#predefined-regexp-expressions

**Comment by gabrielcossette** [comment]
> 
> 
> Faudrait plutôt utiliser les RegExp
> 
> https://www.npmjs.com/package/cspell#predefined-regexp-expressions

Il faudrait effectivement trouver un moyen que le cspell ne s'applique pas aux hyperliens ou sources des images.

Je vais merger pour l'instant et créer une issue séparée.

**Pull request merged**